### PR TITLE
Modal window

### PR DESCRIPTION
**Pull request by ValentinaTenus**
create reusable moal window

### CODE REVIEW

File: src/common/components/modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { useModal } from "~/common/hooks/index";
+
+import styles from "./styles.module.scss";
+
+type ModalProperties = {
+	children: React.ReactNode;
+	footer?: React.ReactNode;
+	isOpen: boolean;
+	onClose: () => void;
+	title: string;
```
Comment (KirillSerg) [inline src/common/components/modal/index.tsx:14]: maybe the name is optional? For example  page 'moderatorx-review' dosen't has this title

File: src/common/components/modal/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
+.modal {
+	align-items: center;
+	background-color: rgba($color: #333, $alpha: 0.6);
```
Comment (KirillSerg) [inline src/common/components/modal/styles.module.scss:3]: maybe you should to add this color to the variables

File: src/common/components/modal/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+	z-index: 100;
+}
+
+.modal__container {
+	background-color: var(--color-white);
+	border-radius: 16px;
+	display: flex;
+	flex-direction: column;
+	font-family: var(--font-family-main);
+	width: 748px;
```
Comment (KirillSerg) [inline src/common/components/modal/styles.module.scss:22]: full width?

**Review by KirillSerg — APPROVED** [review]

File: src/common/components/modal/styles.module.scss (hunk 3 above)
Comment (ValentinaTenus) [reply src/common/components/modal/styles.module.scss:22]: it's for desktop

File: src/common/components/modal/styles.module.scss (hunk 2 above)
Comment (ValentinaTenus) [reply src/common/components/modal/styles.module.scss:3]: I tried, but it doesn't work. Maybe you'll be able to fix it

Comment (KirillSerg) [reply src/common/components/modal/styles.module.scss:3]: i will try, if i don't forget

**Pull request merged**